### PR TITLE
Frontend 1883 - Add Section Sub-menus to Jekyll Menu

### PR DESCRIPTION
**Pull request by langlk**
Resolves web side of [FRONTEND-1883](https://algorithmia.atlassian.net/browse/FRONTEND-1883) and [FRONTEND-1884](https://algorithmia.atlassian.net/browse/FRONTEND-1884)

It turned out to be a bit easier to simply handle both the static and dynamic parts of this at the same time.

**Notes:**
* Relies on styles in https://github.com/algorithmiaio/synapse/pull/190
* Be sure to build vue and synapse files.

### CODE REVIEW

**Comment by bnlb** [comment]
I'm noticing a weird issue where the element with the up/down chevron also disappears when it's clicked, instead of just the `<ul>` within the nested menu. GitHub isn't letting me paste the video here, so I'll send it in Slack.

**Comment by langlk** [comment]
> I'm noticing a weird issue where the element with the up/down chevron also disappears when it's clicked, instead of just the `<ul>` within the nested menu. GitHub isn't letting me paste the video here, so I'll send it in Slack.

Thanks for catching that-- Should be fixed now!

File: _pages/algorithm-development/ci-cd.md [hunk 1]
```diff
@@ -9,6 +9,11 @@ author: jpeck
 image:
   teaser: /language_logos/jenkins.png
 permalink: /algorithm-development/ci-cd/
+menus:
+  algo_development:
+    url: /developers/algorithm-development/ci-cd
+    title: "Deploy Models via Jenkins CI/CD or GitHub Actions"
```
Comment (bnlb) [inline _pages/algorithm-development/ci-cd.md]: Screen Shot 2019-11-19 at 3 48 41 PM
In order to get this whole thing to be visible I need to bump the width of `.syn-page-header-nav` from 15rem to 25rem. Wondering if it'd be simpler to just rename this one to 'CI/CD'

Comment (langlk) [reply _pages/algorithm-development/ci-cd.md]: Huh, that should be wrapping. Do you have synapse updated and built?

Comment (bnlb) [reply _pages/algorithm-development/ci-cd.md]: Yep, I've been nuking the `dist/` folder and rebuilding for each of these PRs. I'll take a look first thing tomorrow to verify.

Comment (langlk) [reply _pages/algorithm-development/ci-cd.md]: Curious!

Comment (langlk) [reply _pages/algorithm-development/ci-cd.md]: @bnlb are you still seeing this?

Comment (bnlb) [reply _pages/algorithm-development/ci-cd.md]: Nope. I see it wrapping now

**Pull request merged**